Standards/scs-XXXX-v1-security-of-iaas-service-software.md

```diff
@@ -0,0 +1,138 @@
+
+## Motivation
+
+On the IaaS Layer the software, that needs to be considered in the scope of this standard, is mainly the APIs of IaaS Services.
+Also there might be shared libraries and other dependencies, that could be considered part of the IaaS Layer.
+In software projects like e.g. OpenStack that provide the main services and all APIs, the software will be modified and receive bug fixes continuously and will receive releases of new versions on a regular basis.
+Older releases will at some point not receive updates anymore, because maintaining more and more releases simultaneously requires too much manpower.
+Thus older versions will also eventually not receive security updates anymore.
+Using versions which do not receive updates anymore threatens the baseline security of deployments and should be avoided under all circumstances.
+
+## Design Considerations
+
+It would be possible to define a minimum version of IaaS Layer software to avoid security risks.
+In the following paragraphs several options of defining a minimum version or dealing with security patches otherwise are discussed.
+
+### Options considered
+
+#### Only Allow the current versions of Software
+
+Considering that OpenStack as one provider of IaaS Layer Software has two releases per year, with one SLURP release per year, this option would require CSPs to update their deployment once or twice a year.
+Updating a whole deployment is a lot of work and requires also good life-cycle management.
+Following only the SLURP releases would reduce this work to once per year.
+
+While following new releases closely already provides a deployment with recent bug fixes and new features, it also makes developing standards easier.
+Differences between releases will accumulate eventually and may render older releases non-compliant to the SCS standards at some point.
+
+On the other hand on the IaaS Level there aren't many breaking changes introduced by releases and also most standards will also work with older releases.
+Security Updates and Bug fixes are also provided by OpenStack for a few older releases with the state `maintained`.
+Additionally the [SCS reference implementation](https://github.com/SovereignCloudStack/release-notes/blob/main/Release7.md) is integrating OpenStack releases after half a year - so about the time when a new release is cut by OpenStack.
+Considering a CSP that wants to use only SLURP releases and waiting for the reference implementation will already be over a year or 2 releases behind, which cannot be considered as using the current version of IaaS Layer Software.
+Thus this option can be discarded.
+
+#### Allow only maintained versions of Software
+
+While following closely to the newest releases could be advised, there are several downsides to requiring this workflow, even if it would be only for SLURP releases.
+Following the SCS reference implementation for example would also lead into being a little bit behind the newest release.
```

Comment:
```suggestion
Following the SCS reference implementation for example would also lead into being a little bit behind the newest OpenStack release.
```

Just adding this to make it clear that not the SCS reference implementation release is meant in the second part here.